modules/stanford_profile_styles/lib/scss/component/paragraph/card/_card.scss

```diff
@@ -0,0 +1,21 @@
+@charset 'UTF-8';
+
+.su-card {
+  .su-card__contents {
+    h2 {
+      @include type-c;
+    }
+
+    h3 {
+      font-size: 1.4em;
```

Comment:
I don't see all the styling that is requrested. According to the ticket requirements:

> Heading Three: h3 should display in 1.40em. Approximate pixel size per breakpoint;
    XS-SM: type 2.3rem // 23px
    MD-LG-XL: 2.5rem // 25px
    XXL: 3.4rem // 34px